static/js/tools/timeline_util.ts

```diff
@@ -200,40 +205,50 @@ function getStatsVarInfo(dcids: string[]) {
   });
 }
 
-function getStatsVar(dcids: string[]) {
-  if(dcids.length === 0) {
-  return Promise.resolve(new Set<string>());
-}
-const promises = [];
-// ToDo: read the set of statsvars available for multiple dcids from server side
-for (const dcid of dcids) {
-  promises.push(
-    axios.get("/api/place/statsvars/" + dcid).then((resp) => {
-      return resp.data;
-    })
-  );
+function getPC() {
+  const vars = getUrlVars() as VarUrl;;
+  if ("pc" in vars) {
+    return vars.pc === "1";
+  } else {
+    return false;
+  }
 }
-return Promise.all(promises).then((values) => {
-  let statvars = new Set(); // Count_Person not in List ???
-  for (const value of values) {
-    statvars = new Set([...Array.from(statvars), ...value])
-  }
-  Object.keys(MAPPING).map((key) => {
-    if (statvars.has(MAPPING[key])) {
-      statvars.add(key);
+function getStatsVar(dcids: string[]) {
+  if (dcids.length === 0) {
+    return Promise.resolve(new Set<string>());
+  }
+  const promises = [];
+  // ToDo: read the set of statsvars available for multiple dcids from server side
+  for (const dcid of dcids) {
+    promises.push(
+      axios.get("/api/place/statsvars/" + dcid).then((resp) => {
+        return resp.data;
+      })
+    );
+  }
+  return Promise.all(promises).then((values) => {
+    let statvars = new Set(); // Count_Person not in List ???
+    for (const value of values) {
+      statvars = new Set([...Array.from(statvars), ...value])
     }
-  })
-  return statvars;
-}) as Promise<Set<string>>;
+    Object.keys(MAPPING).map((key) => {
+      if (statvars.has(MAPPING[key])) {
+        statvars.add(key);
+      }
+    })
+    return statvars;
+  }) as Promise<Set<string>>;
 }
 
 export {
   updateUrlStatsVar,
   updateUrlPlace,
+  updateUrlPC,
   parseStatVarPath,
   parsePlace,
   getStatsVarInfo,
   getPlaceNames,
+  getPC,
```

Comment:
same to combine 3 get functions into one

Reply:
done.